cli/bpmetadata/int-test/workflow.sh

```diff
@@ -34,7 +34,11 @@ fi
 
 mkdir $WORKING_FOLDER && pushd $WORKING_FOLDER
 kpt pkg get https://github.com/terraform-google-modules/terraform-google-cloud-storage.git/@v4.0.0 "./$BLUPRINT_FOLDER/"
```

Comment:
It would also be good to add a few comments of what the int-test is doing.

Reply:
> I'm not familiar with the intention, but do we want to use kpt to get a particular package version and then use git against HEAD?

That's technically what I'm doing but I'm not familiar with how to do that with just one kpt command, if that's what you're suggesting. AFAIK, `kpt pkg get` does not maintain git info like `git clone` does which is why I'm going a `git init` and `git remote...` right after. If I'm missing something, please correct me.

I've added comments to explain the intention in a bit more detail.

Reply:
Hmm - Likely because `kpt pkg` don't have to map 1:1 to a repo, but can also be a repo's subdirectory.  Would it work to simply clone the repo at the desired version? `git clone -b v4.0.0 https://github.com/terraform-google-modules/terraform-google-cloud-storage.git`

Reply:
Thanks for the tip on `git clone`. I've modified the script to use that instead.